### PR TITLE
VIITE-1324 Remove OTH code from VIITE

### PR DESCRIPTION
**Pull request by skosonen**
Moved viite-UI under UI.
Removed OTH code that was not in use in VIITE.

### CODE REVIEW

**Comment by Campanha** [comment]
damn =o

**Comment by skosonen** [comment]
Now this pull request is much easier to review since I renamed UI back to viite-UI. We can rename it to UI later if we want.

**Comment by pedrosag** [comment]
That is wierding me out, that rename and Git suddenly shrinks back 10K additions and 10K removals how?

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/Sequences.scala [hunk 1]
```diff
@@ -5,23 +5,23 @@ import slick.driver.JdbcDriver.backend.Database
 import Database.dynamicSession
 
 object Sequences {
-  def nextPrimaryKeySeqValue = {
+  def nextPrimaryKeySeqValue: Long = {
```
Comment (leirasf) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/Sequences.scala]: I think that sequence is no longer used Viite, or we should have only one.

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/Sequences.scala [hunk 2]
```diff
@@ -5,23 +5,23 @@ import slick.driver.JdbcDriver.backend.Database
...
     nextPrimaryKeyId.as[Long].first
   }
 
-  def nextLrmPositionPrimaryKeySeqValue = {
+  def nextLrmPositionPrimaryKeySeqValue: Long = {
```
Comment (leirasf) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/Sequences.scala]: I think that sequence is no longer used in viite, or we should have only one.

File: digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/authentication/SessionApi.scala [hunk 3]
```diff
@@ -30,6 +30,7 @@ class SessionApi extends ScalatraServlet {
       case Some(u) => {
         logger.info("User exists: " + username + "(" + u + ")")
         BadRequest("User exists")
+        // TODO Should we allow updating of the existing user and call userProvider.saveUser(u)?
```
Comment (leirasf) [inline digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/authentication/SessionApi.scala]: I'm thinking if it should be possible to create a user on the sessionAPI, probably it should be on the UserConfigurationApi.

Comment (skosonen) [reply digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/authentication/SessionApi.scala]: Yes, it is a bit strange that there was this kind of functionality in SessionApi.

**Comment by skosonen** [comment]
I run tests locally after a fixture reset and **all tests passed**.

**Comment by skosonen** [comment]
When building master with the same Jenkins job, it fails the same way:
`[info] Resolving org.apache.geronimo.specs#geronimo-jta_1.1_spec;1.1.1 ...
[error] Server access Error: Connection timed out (Connection timed out) url=http://repo.typesafe.com/typesafe/ivy-releases/org.apache.geronimo.specs/geronimo-jta_1.1_spec/1.1.1/ivys/ivy.xml
[error] Server access Error: Connection timed out (Connection timed out) url=http://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/org.apache.geronimo.specs/geronimo-jta_1.1_spec/1.1.1/ivys/ivy.xml
[error] Server access Error: Connection timed out (Connection timed out) url=http://repo1.maven.org/maven2/org/apache/geronimo/specs/geronimo-jta_1.1_spec/1.1.1/geronimo-jta_1.1_spec-1.1.1.pom
[warn] 	module not found: org.apache.geronimo.specs#geronimo-jta_1.1_spec;1.1.1
[warn] ==== typesafe-ivy-releases: tried
[warn]   http://repo.typesafe.com/typesafe/ivy-releases/org.apache.geronimo.specs/geronimo-jta_1.1_spec/1.1.1/ivys/ivy.xml
[warn] ==== sbt-plugin-releases: tried
[warn]   http://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/org.apache.geronimo.specs/geronimo-jta_1.1_spec/1.1.1/ivys/ivy.xml
[warn] ==== local: tried
[warn]   /var/lib/jenkins/.ivy2/local/org.apache.geronimo.specs/geronimo-jta_1.1_spec/1.1.1/ivys/ivy.xml
[warn] ==== public: tried
[warn]   http://repo1.maven.org/maven2/org/apache/geronimo/specs/geronimo-jta_1.1_spec/1.1.1/geronimo-jta_1.1_spec-1.1.1.pom
...
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.apache.geronimo.specs#geronimo-jta_1.1_spec;1.1.1: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
sbt.ResolveException: unresolved dependency: org.apache.geronimo.specs#geronimo-jta_1.1_spec;1.1.1: not found
	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:217)
...
	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:81)
	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:102)
...
	at java.lang.Thread.run(Thread.java:748)
[error] (*:update) sbt.ResolveException: unresolved dependency: org.apache.geronimo.specs#geronimo-jta_1.1_spec;1.1.1: not found
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? Build step 'Build using sbt' changed build result to FAILURE
Build step 'Build using sbt' marked build as failure
Finished: FAILURE`

**Pull request merged**